frontend/src/views/Login.vue

```diff
@@ -13,7 +13,7 @@
 					<form class="flex flex-col space-y-4" @submit.prevent="submit">
 						<Input
 							:label="__('Email')"
-							placeholder="johndoe@mail.com"
+							:placeholder="__('johndoe@mail.com')"
```

Comment:
We don't need to translate this IG?

Reply:
In Germany, nobody knows John Doe. He's called Max Mustermann here. :D